br/pkg/restore/split.go

```diff
@@ -481,10 +428,19 @@ func (rs *RegionSplitter) WaitForScatterRegionsTimeout(ctx context.Context, regi
 		reScatterRegions = make([]*split.RegionInfo, 0, len(regionInfos))
 	)
 	for {
-		ctx1 := context.WithValue(ctx, retryTimes, retryCnt)
+		loggedLongRetry := false
 		reScatterRegions = reScatterRegions[:0]
 		for regionID, regionInfo := range leftRegions {
-			ok, rescatter, err := rs.isScatterRegionFinished(ctx1, regionID)
+			if retryCnt > 10 && !loggedLongRetry {
```

Comment:
maybe use a ratelimiter of burst 1 to log it every xx seconds

Reply:
the old logic is worse... it will print all regions in one retry. And there's a sleep in line 476